swift/Samples/TicTacToe/Sources/Game/RunGameWorkflow.swift

```diff
@@ -115,11 +124,40 @@ extension RunGameWorkflow {
                     leftItem: BackStackScreen.BarContent.BarButtonItem.button(BackStackScreen.BarContent.Button(
                         content: .text("Quit"),
                         handler: {
-                            sink.send(.back)
+                            sink.send(.confirmQuit)
+                        }))))))
```

Comment:
Nit: Generally, I'd put these closing parenthesis on new lines and match the indentation. Not sure if I'm the only one that feels this way.

Reply:
Yes, will fix that. My brain is still getting rewind into thinking in Swift standards. Its in the hairball stage right now :)